### PR TITLE
Feature/6 add docker support

### PR DESCRIPTION
**Pull request by OlegMikus**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+RUN pip install --upgrade pip
+RUN pip3 install pipenv
+
+COPY Pipfile Pipfile.lock ./
+
+RUN pipenv install --system --deploy --ignore-pipfile
+COPY . .
```
Comment (ishaurko) [inline Dockerfile:14]: copy only src

**Review by ishaurko — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (OlegMikus) [reply Dockerfile:14]: I also need .env for everything to work

File: src/models/migrations/README [hunk 2]
```diff
@@ -0,0 +1 @@
+Single-database configuration for Flask.
```
Comment (ishaurko) [inline src/models/migrations/README:1]: Pretty unhelpful readme

**Review by ishaurko — CHANGES_REQUESTED** [review]

File: src/models/migrations/README (hunk 2 above)
Comment (OlegMikus) [reply src/models/migrations/README:1]: I'll remove migrations from this PR

File: .env [hunk 3]
```diff
@@ -1,7 +1,7 @@
 SECRET_KEY=sytjhrestjhwesr65w465hyer465y
 DATABASE_PASSWORD=password
 DATABASE_USER=postgres
-DATABASE_HOST=localhost:5432
+DATABASE_HOST=localhost:54322
```
[the remark this answers is not in the document]
Comment (OlegMikus) [reply .env:4]: Ok

**Review by ishaurko — APPROVED** [review]

**Pull request merged**